.github/workflows/dependency-review.yml

```
@@ -0,0 +1,19 @@
+name: 'Dependency Review'
+on:
+  pull_request:
+    branches: ['master']
```

```suggestion
    branches: ['develop']
```

this is our main branch (we use gitflow, master is the currently released branch)